agreements/expenses-agreement.md

```diff
@@ -0,0 +1,44 @@
+# Expenses Agreement
+
+As a worker-owned business Root Systems members all have the ability to claim expenses relating to the operation of the business. 
+
+## This may include:
+
+ - Technical equipment (laptops, keyboards, etc...)
+ - Office equipment (coffee machine, chairs, desk, lights, etc...)
+ - Office space rent
+ - Retreats and Away days (travel, food and accomadation)
+ - Event expenses (tickets, foods, etc...)
+ - Every day business lunches (50% tax deductable for members, 100% for guests)
+ - Travel expenses (transport, hotels, flights, etc...)
+ - Saas tools (google, toggl, loomio, xero, etc...)
+ - Memberships (Enspiral)
+
+## Accounting for expenses
+
+Expenses fall into two categories:
+
+ - Team expense
+ - Member expense
+
+### Team expense
+
+These are expenses that are accounted for out of core funds and facilitated by the Coordinators. The list at the bottom of this agreement defines ongoing Team expenses that we do not need raise a decision for. Beyond this any member can raise a proposal via Loomio to add another Team expense (one-off or on-going). Team expenses will be accounted for out of our opperational budget in Xero.
+
+### Member expense
+
+Any Member can expense off any items they identify as being a business expense. Coordinators will facilitate this to provide support and understanding of what constitutes a business expense. Member expenses will be accounted for through deducting the expense minus GST from their internal account.
+
+**Everyday business lunch expenses will be standardised to $20/day to make internal accounting easier.
```

Comment:
this feels like a _lot_ of money to spend on lunch, maybe it's not. is lunch a Team expense or a Member expense? if a Team expense, it feels like this will create the same FOMO (Fear Of Missing Out) vibe i mentioned before where everyone wants to spend as much team money as is possible every day because otherwise they miss out on "free money". also if a Team expense, it's in the wrong section, since this is in the Member section.